roles/ngi_reports/tasks/main.yml

```diff
@@ -0,0 +1,60 @@
+---
+#- name: Check if dev_IS virtual env exists 
```

Comment:
Might these uncommented tasks be needed in the future or should they be removed now?